components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt.ui/src/main/resources/web/identity-claim-mgt/add-local-claim.jsp

```diff
@@ -98,6 +167,22 @@
 
             <script type="text/javascript">
 
+                var claimData = [];
+                <%
+                       for (int i = 0; i < localClaims.length; i++) {
+                            if (!localClaims[i].getLocalClaimURI().equals(localClaimURI)) {
+
+                                AttributeMappingDTO[] attributeMap = localClaims[i].getAttributeMappings();
+                                for (int j = 0; j < attributeMap.length; j++) {
+                                    %>
+                                    claimData.push("<%=Encode.forJavaScript(localClaims[i].getLocalClaimURI())
```

Comment:
Can't we store mapped attributes in a multidimensional array, rather using string manipulation
`mappedAttributes[ClaimURI][UserDomain]=MappedAttribute`